packages/manager/modules/pci/src/projects/project/private-registry/components/plan-upgrade/RegistryPlan.class.js

```diff
@@ -0,0 +1,37 @@
+import get from 'lodash/get';
+import set from 'lodash/set';
+
+export default class RegistyPlan {
+  constructor(resource) {
```

Comment:
Could you define the property list instead of `resource` to know which properties are expected ?